core/src/test/java/org/axonframework/deadline/quartz/DeadlineJobDataBinderTest.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.deadline.quartz;
+
+import org.axonframework.commandhandling.model.AggregateScopeDescriptor;
+import org.axonframework.deadline.DeadlineMessage;
+import org.axonframework.deadline.GenericDeadlineMessage;
+import org.axonframework.messaging.MetaData;
+import org.axonframework.messaging.ScopeDescriptor;
+import org.axonframework.serialization.JavaSerializer;
+import org.axonframework.serialization.SerializedType;
+import org.axonframework.serialization.Serializer;
+import org.axonframework.serialization.SimpleSerializedObject;
+import org.axonframework.serialization.json.JacksonSerializer;
+import org.axonframework.serialization.xml.XStreamSerializer;
+import org.junit.*;
+import org.mockito.*;
+import org.quartz.JobDataMap;
+
+import java.util.function.Predicate;
+
+import static org.axonframework.deadline.quartz.DeadlineJob.DeadlineJobDataBinder.*;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class DeadlineJobDataBinderTest {
```

Comment:
Could this be a Parameterized test where parameter would be a serializer? We could cut off test cases by a third.

Reply:
Done, kinda like the outcome. Interested in your feedback on it though!

Reply:
Looks pretty good to me!